src/Search/Console.php

```diff
@@ -258,6 +258,7 @@ public function __construct(Application $console, Kernel $kernel)
         // Some annotations
         Register::registerLoader();
 
+        // TODO: remove when it is *never* passed in by the formula or anything else
         $this->console->getDefinition()->addOption(new InputOption('--env', '-e', InputOption::VALUE_OPTIONAL, 'The Environment name. Deprecated and not used', 'dev'));
```

Comment:
> Deprecated and not used

Which is it? 😅

Reply:
Deprecated to pass it in, because it is not used (but would explode if you pass it in and this definition is not here, IIRC)

Reply:
Deprecation preserves BC. Though maybe it was never used anyway.